### PR TITLE
Added more text for translation

### PR DESCRIPTION
**Pull request by neoatomic**

### CODE REVIEW

**Comment by SickRage2** [comment]
Thanks for the pull request! Before a real human comes by, please make sure your PR has all of the below criteria checked
- [ ] Give a description on what the PR is for.
- [x] Make sure your PR is based on the DEVELOP branch
- [x] Don't send big changes all at once. Split up big PRs into multiple smaller PRs that are easier to manage and review

Please make sure you also read [contribution guide](https://github.com/SickRage/SickRage/blob/master/contributing.md) and followed all of the steps.

Thanks!

File: sickbeard/versionChecker.py [hunk 1]
```diff
@@ -90,23 +90,23 @@ def run_backup_if_safe(self):
...
+        ui.notifications.message(_('Backup', 'Config backup in progress...'))
         try:
             backupDir = ek(os.path.join, sickbeard.DATA_DIR, 'backup')
             if not ek(os.path.isdir, backupDir):
                 ek(os.mkdir, backupDir)
 
             if self._keeplatestbackup(backupDir) and self._backup(backupDir):
                 logger.log(u"Config backup successful, updating...")
-                ui.notifications.message('Backup', 'Config backup successful, updating...')
+                ui.notifications.message(_('Backup', 'Config backup successful, updating...'))
```
Comment (miigotu) [inline sickbeard/versionChecker.py]: 'Backup' and 'Config backup successful, updating...' are 2 different strings. You have to wrap each individually

File: sickbeard/versionChecker.py [hunk 2]
```diff
@@ -90,23 +90,23 @@ def run_backup_if_safe(self):
     def _runbackup(self):
         # Do a system backup before update
         logger.log(u"Config backup in progress...")
-        ui.notifications.message('Backup', 'Config backup in progress...')
+        ui.notifications.message(_('Backup)', '(_Config backup in progress...'))
```
Comment (miigotu) [inline sickbeard/versionChecker.py]: ^^^

File: sickbeard/versionChecker.py [hunk 3]
```diff
@@ -90,23 +90,23 @@ def run_backup_if_safe(self):
     def _runbackup(self):
         # Do a system backup before update
         logger.log(u"Config backup in progress...")
-        ui.notifications.message('Backup', 'Config backup in progress...')
+        ui.notifications.message(_('Backup'), (_'Config backup in progress...'))
```
Comment (miigotu) [inline sickbeard/versionChecker.py]: `ui.notifications.message(_('Backup'), _('Config backup in progress...'))`

**Pull request merged**